tests/test_tululbot.py

```diff
@@ -207,40 +243,37 @@ def test_leli_command_with_multiword_term(app):
                 }
             }
         }
-        app.post('/', data=json.dumps(payload),
-                 content_type='application/json')
+        do_post(client, payload)
 
-        mock_requests.get.assert_called_once_with(
-            'https://en.wikipedia.org/w/index.php?search=waza+und+technik'
-        )
+        mock_leli.assert_called_once_with('waza und technik')
 
 
-def test_quote(app):
-    payload = {
-        'update_id': 12345,
-        'message': {
-            'message_id': 100,
-            'text': '/quote',
-            'chat': {
-                'id': 1
+def test_quote(client):
+    with patch('tululbot.QuoteEngine') as mock_engine:
+        mock_engine.return_value.retrieve_random.return_value = 'some random quote'
+        payload = {
+            'update_id': 12345,
+            'message': {
+                'message_id': 100,
+                'text': '/quote',
+                'chat': {
+                    'id': 1
+                }
             }
         }
-    }
 
-    rv = app.post('/', data=json.dumps(payload),
-                  content_type='application/json')
+        rv = do_post(client, payload)
 
-    assert rv.status_code == 200
-    json_response = json.loads(rv.get_data(as_text=True))
-    assert json_response['method'] == 'sendMessage'
-    assert json_response['chat_id'] == 1
-    # Fix the test later, baby
-    # assert json_response['text'] == expected_text
-    assert json_response['disable_web_page_preview'] == 'false'
-    assert json_response['reply_to_message_id'] == 100
+        assert rv.status_code == 200
```

Comment:
@fushar Oops I missed this. Thanks for noticing.
@wazaundtechnik Yes. Python knows how to compare dictionary. Try it yourself.